Test/AwsCommonRuntimeKitTests/crt/UtilityTests.swift

```diff
@@ -22,5 +22,19 @@ class UtilityTests: XCTestCase {
         
         XCTAssertEqual(md5, "iB0/3YSo7maijL0IGOgA9g==")
     }
+    
+    func testSha256() throws {
+        let hello = "Hello"
+        let sha256 = hello.base64EncodedSha256()
+        XCTAssertEqual(sha256, "GF+NsyJx/iX1Yab8k4suJkMG7DBO2lGAB9F2SCY4GWk=")
```

Comment:
probably worth testing a sha of the empty string "" -> "e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855" since it's edge-case-y

Reply:
added a test with empty string